requirements.txt

```diff
@@ -1,12 +1,8 @@
 docopt==0.6.2
-dohq-artifactory==0.4.112; python_version < '3.5'
-dohq-artifactory>=0.7.377; python_version >= '3.5'
-Jinja2<2.11; python_version < '3.5'
-Jinja2>=2.11; python_version >= '3.5'
+dohq-artifactory>=0.8.3
+Jinja2>=2.11
 patool==1.12
 pyunpack==0.2
-PyYAML<5.2; python_version < '3.5'
-PyYAML>=5.2; python_version >= '3.5'
-requests<2.22; python_version < '3.5'
-requests>=2.22; python_version >= '3.5'
-urllib3==1.24.3
+PyYAML>=5.2,<6.0
+requests>=2.25.1,<3.0.0
+urllib3<1.25,>=1.21.1
```

Comment:
Let's check the possibility of using `urllib3>=1.26.5`. Create separate issue for this